text_extensions_for_pandas/io/conll.py

```diff
@@ -40,6 +40,7 @@
 # Special token that CoNLL-2003 format uses to delineate the documents in
 # the collection.
 _CONLL_DOC_SEPARATOR = "-DOCSTART-"
+_EWT_DOC_SEPERATOR = "# newdoc id"
```

Comment:
I think this separator is actually officially part of the CoNLL-U standard and isn't just for EWT; see https://universaldependencies.org/format.html#paragraph-and-document-boundaries